BootloaderCommonPkg/Library/XhciLib/XhcPeim.h

```diff
@@ -20,9 +20,10 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 #include <Library/TimerLib.h>
 #include <Library/IoLib.h>
 #include <Library/MemoryAllocationLib.h>
+#include <Library/IoMmuLib.h>
 
-#define  PeiServicesAllocatePages(a, b, c)   \
-         (((*(c) = (UINTN)AllocatePages (b)) != 0) ? EFI_SUCCESS : EFI_OUT_OF_RESOURCES)
+#undef   DEBUG_INFO
+#define  DEBUG_INFO   DEBUG_VERBOSE
 

```

Comment:
Is this for debug purpose? I saw several places changing code to DEBUG_INFO from DEBUG_VERBOSE

Reply:
Ahh, yes. It was for debug and should be removed.  Good catch!

Reply:
Most of the code was directly synced from EDK2.

Reply:
sorry, I forgot that I added these code to remove too much debug message in XHCI driver.  I did not try to change the debug level one by one since that is the original EDK2 code.  To address this, we can refine the INFO to VERBOSE level.  And only show error or warning messages.